formbricks/formbricks
# fix: color coding in NPS misplaces text

## What does this PR do?
Fixed color coding in NPS misplaces text 
Fixes #2922 


Before 
<img width="529" alt="353065709-f4fa8193-2b16-4f8c-bf51-34c18b978908" src="https://github.com/user-attachments/assets/d0c92d4e-7d7e-4e4c-8053-f299862334f1">

Now
![image](https://github.com/user-attachments/assets/f1ce8929-00c0-45c0-b538-57906c7c366b)
![image](https://github.com/user-attachments/assets/758c1827-75ad-4c68-aed6-f88e7b06de99)

## How should this be tested?



- Create a survey with an NPS question
- Toggle "add color code" on / off
- Observe the recommended rating UI 

## Checklist



### Required


### Appreciated
